service/dap/server.go

```diff
@@ -1567,7 +1566,39 @@ func (s *Server) onCancelRequest(request *dap.CancelRequest) {
 // Capability 'supportsExceptionInfoRequest' is set 'initialize' response.
 func (s *Server) onExceptionInfoRequest(request *dap.ExceptionInfoRequest) {
 	goroutineID := request.Arguments.ThreadId
-	body := s.exceptions[goroutineID]
+	body := dap.ExceptionInfoResponseBody{}
+	// Get the thread for the goroutine and the current state.
+	thread, _ := s.debugger.FindGoroutine(goroutineID)
```

Comment:
Can you call this variable `g`? We're using `thread` to mean "OS thread" everywhere else.

Reply:
Done.

Reply:
Sorry I didn't notice that this actually *is* a Thread. Please change FindGoroutine to return a goroutine.